app/proto/events.proto

```diff
@@ -280,12 +280,14 @@ message ListMyEventsReq {
 
   uint32 page_size = 5;
   string page_token = 6;
+  uint32 page_number = 9;
```

Comment:
Please change to

```
oneof pagination {
  string page_token = 6;
  uint32 page_number = 9;
}
```

aka just wrap both inside the `oneof pagination`

Reply:
Okay just pushed this! Had to adjust the events.py and seach.py a bit for the oneof so the tests would still pass :-)